core/src/navigation/LeftNav.html

```diff
@@ -139,6 +139,13 @@
     {/if} {/each}
   </nav>
   {/if}
+  {#if isFooterVisible(footerText)}
+  <div class="lui-sideNavFooter fd-has-padding-small fd-has-type-minus-1">
+    {#if footerText}
```

Comment:
Do we need the container at all, if `footerText` is undefined? else I would put it before `isFooterVisible`.

Reply:
I did the implementation (this template + `isFooterVisible` function) with having the future placement of SideNav close button on mind. It's visible on Zeplin. Lydia says it's a concept and likely be introduced in the future. 
![image](https://user-images.githubusercontent.com/22825433/56964776-25643c00-6b5c-11e9-886b-d70178081b33.png)

Furthermore, who knows - maybe we'll put there something else in the future? 
In general, I'm against useless containers (!) but this is a special case in my opinion.